### PR TITLE
Add search bar in plannify; remove footer in planner component

### PR DESCRIPTION
**Pull request by bofeiw**
- search bar copied from pathways is added to plannify, it simply log the retrieved information when a course is selected.
- footer is removed from planner component because it is redundant and causing positioning issues. it is supposed to be outside of panner component (which is the case for now).

### CODE REVIEW

**Comment by bofeiw** [comment]
Lol not sure why all these commits (which are already merged) are shown here after merging from master

File: client/src/components/planner/DegreePlanner.js [hunk 1]
```diff
@@ -1,10 +1,65 @@
...
+        margin: "auto",
+        paddingTop: "2%"
+    }
+});
 
 export default function DegreePlanner() {
+    const classes = useStyles();
+
+    const [course, setCourse] = useState({});
+    const [prereqs, setPrereqs] = useState([]);
```
Comment (jjccharles) [inline client/src/components/planner/DegreePlanner.js]: For the sake of processing and component complexity, lets remove the prereq and postreq logic so for now it just calls upon course endpoint and logs selected courses.

**Review by jjccharles — CHANGES_REQUESTED** [review]
Really good stuff man! This will be useful to work towards the actual degree planning components :D

File: client/src/components/planner/DegreePlanner.js (hunk 1 above)
Comment (bofeiw) [reply client/src/components/planner/DegreePlanner.js]: prereq and postreq logic removed

**Comment by bofeiw** [comment]
@jcharles8246 prereq and postreq logic removed

**Review by jjccharles — APPROVED** [review]

**Pull request merged**